### DISCUSSION TITLE
Speaker Data page using same numbers?

### DISCUSSION THREAD

**Discussion by MatiasERP** [pierreaubert/spinorama#268] [General]
I think there is a bug in the individual speaker page, under the Data tab, where the metrics are shown. I think it is using only the latest  or the Default measurements to calculate them, and not the specific measurement I chose on that page.

Example:
Sointuva AWG - Erin's
https://www.spinorama.org/speakers/March%20Audio%20Sointuva%20AWG/ErinsAudioCorner/index_eac.html
Sointuva AWG - vendor's
https://www.spinorama.org/speakers/March%20Audio%20Sointuva%20AWG/March%20Audio/index_vendor.html

Sierra 1 V2 - Erin's
https://www.spinorama.org/speakers/Ascend%20Acoustics%20Sierra%201%20V2/ErinsAudioCorner/index_eac.html
Sierra 1 V2 - vendor's
https://www.spinorama.org/speakers/Ascend%20Acoustics%20Sierra%201%20V2/Ascend%20Acoustics/index_vendor.html
Sierra 1 V2 - ASR's
https://www.spinorama.org/speakers/Ascend%20Acoustics%20Sierra%201%20V2/ASR/index_asr.html

All the Data tabs have the exact same numbers between each group.

**Comment by pierreaubert** [comment]
That's a bug. Thanks for spotting it. Code is corrected but not yet pushed.
It will take a few days.

On Mon, Apr 22, 2024 at 3:51 PM MatiasERP ***@***.***> wrote:

> I think there is a bug in the individual speaker page, under the Data tab,
> where the metrics are shown. I think it is using only the latest or the
> Default measurements to calculate them, and not the specific measurement I
> chose on that page.
>
> Example:
> Sointuva AWG - Erin's
>
> https://www.spinorama.org/speakers/March%20Audio%20Sointuva%20AWG/ErinsAudioCorner/index_eac.html
> Sointuva AWG - vendor's
>
> https://www.spinorama.org/speakers/March%20Audio%20Sointuva%20AWG/March%20Audio/index_vendor.html
>
> Sierra 1 V2 - Erin's
>
> https://www.spinorama.org/speakers/Ascend%20Acoustics%20Sierra%201%20V2/ErinsAudioCorner/index_eac.html
> Sierra 1 V2 - vendor's
>
> https://www.spinorama.org/speakers/Ascend%20Acoustics%20Sierra%201%20V2/Ascend%20Acoustics/index_vendor.html
> Sierra 1 V2 - ASR's
>
> https://www.spinorama.org/speakers/Ascend%20Acoustics%20Sierra%201%20V2/ASR/index_asr.html
>
> All the Data tabs have the exact same numbers between each group.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pierreaubert/spinorama/discussions/268>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANBKHZY74CUEKCRQETMAATY6UIX7AVCNFSM6AAAAABGS2ULPOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGU2DGOJVGU>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--
PfA
